### DISCUSSION TITLE
Time based publication

### DISCUSSION THREAD

**Discussion by TommsNL** [liedekef/events-made-easy#506] [Q&A] [answered]
Hi,

I'd like to publish my Event/RSVP form at a certain date & time. I have it private at the moment so only event organizers can use the form to register. But on a certain date & time I'd like to publish the event so the general public can register.

Is there any efficient way to achieve this?

I could publish a Wordpress page on a pre defined date and time with the shortcode to the event. But the event needs to be public to work. I'm afraid people will still find the event and register...

**Comment by liedekef** [comment, marked as answer]
You can make the event "hidden" (that way it doesn't appear in lists) and also limit from when people can rsvp for the event
